E01-Geometry/ramas/src/Geometry/Geometry.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kilo\Geometry;
+
+use Kilo\Geometry\Shape\ShapeInterface;
+
+class Geometry
+{
+    private $shape;
+
+    public function __construct(ShapeInterface $shape)
```

Comment:
By introducing `ShapeInterface` into `Geometry` dependency, you've made this class stateful.
Now we would need a new instance of `Geometry` every time we would want to calculate area.

Geometry doesn't have `has a` relationship with `Shape`.
`Shape` is a target as opposed to a dependency.

Move `Shape` into parameter of `area` and `perimeter`.